src/pages/Dashboard/components/ProfileAvatar/AvatarSelectModal.tsx

```diff
@@ -75,34 +80,38 @@ export const AvatarSelectModal: React.FC<AvatarSelectModalProps> = props => {
 
   const defaultValue = selectedNftAvatar ?? defaultWalletImage
 
-  const { getRootProps, getRadioProps } = useRadioGroup({
+  const { getRootProps, getRadioProps, setValue } = useRadioGroup({
     name: 'framework',
     onChange: setSelected,
     defaultValue,
   })
 
   const group = getRootProps()
 
-  const renderItems = useMemo(() => {
-    return filteredNfts?.map(({ assetId, medias }) => {
-      if (!assetId) return null
-      const mediaUrl = medias?.[0]?.originalUrl
-      return <AvatarRadio key={assetId} src={mediaUrl} {...getRadioProps({ value: assetId })} />
-    })
-  }, [filteredNfts, getRadioProps])
+  const handleSaveChanges = useCallback(
+    (selectedAvatar: string | null) => {
```

Comment:
Can this still be null? Since this will now be the default image, or the selected AssetId

Reply:
selected can start out as null if they have not made a selection yet.